server/homebrew.api.js

```diff
@@ -234,30 +234,31 @@ const updateBrew = async (req, res)=>{
 	if(req.account) {
 		brew.authors = _.uniq(_.concat(brew.authors, req.account.username));
 	}
-
-	// Fetch the brew from the database again (if it existed there to begin with), and assign the existing brew to it
-	brew = _.assign(await HomebrewModel.findOne({ _id: brew._id }), brew);
-
-	if(!brew.markModified) {
-		// If it wasn't in the database, create a new db brew
-		brew = new HomebrewModel(brew);
+	// we need the tag type change in both getBrew and here to handle the case where we don't have a stub on which to perform the modification
```

Comment:
When would this ever occur? The only case I can think of is opening a Google Brew that was never stubbed. In that case, we have never saved tags to Google Brews (stubs came before tags we implemented). I propose instead we modify the `getGoogleBrew()` function in `googleActions.js` so it just.... doesn't try to retrieve the tags field. Then we don't have to worry about doing this conversion here.

Reply:
Ah that's a good point! Yeah I'll make that change, good call.